serbian/alkohol/alkohol.md

```diff
@@ -0,0 +1,2 @@
+Vekovima optuzivana supstanca za drustveno neprihvatljivo ponasanje. To jutro kao i prethodno posle [paba](https://en.wikipedia.org/wiki/Pub) svratio je u pekaru, nasmejao se i [upitao je nesto](../burekSir/noz/noz.md)
```

Comment:
This line contains more than one link.  Please put links on separate lines with a blank line in between so that options will be visually separated.